tracee-rules/Makefile

```diff
@@ -6,10 +6,10 @@ OUT_BIN := $(OUT_DIR)/tracee-rules
 OUT_RULES := $(OUT_DIR)/rules
 GO_SRC := $(shell find . -type f -name '*.go')
 GOSIGNATURES_DIR ?= signatures/golang
-GOSIGNATURES_SRC := $(shell find $(GOSIGNATURES_DIR) -type f -name '*.go' ! -name '*_test.go')
+GOSIGNATURES_SRC := $(shell find $(GOSIGNATURES_DIR) -type f -name '*.go' ! -name '*_test.go' ! -path '*examples*')
```

Comment:
small one, but perhaps it'll be safer to change this to `$(GOSIGNATURES)/examples/*` same for the rego one. not a stopper though, whatever you think

Reply:
Yeah why not - I added your suggestion.